web-lib-funcs.pl

```diff
@@ -266,6 +266,31 @@ sub html_escape
 return $tmp;
 }
 
+=head2 html_unescape(string)
+
+Converts HTML entities to the corresponding character
+
+=cut
+sub html_unescape
+{
+my ($str) = @_;
+if (!defined $str) {
+    return ''; # empty string
```

Comment:
Indent looks messed up?

Reply:
> Indent looks messed up?

Yes, it is. Fixed here https://github.com/webmin/webmin/commit/3d67e17cef426759eebc9a545727acad18eeeb8c.